### PR TITLE
Add Contentful Bootstrap Tutorial

### PR DESCRIPTION
**Pull request by dlitvakb**

### CODE REVIEW

File: docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+##### Pre-Built Templates
+
+For the case of pre-built templates, we have:
+
+* `blog`
+* `catalogue`
+* `gallery`
+
+Each of this templates come bundled with a few Content Types and Entries.
```
Comment (grncdr) [inline docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: each of _these_ templates

File: docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+They are useful for small demos, but are definitely not production ready spaces.
```
Comment (grncdr) [inline docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: The spaces set up by predefined templates theoretically could be used for a production site, but the reality is that everybody has custom needs and one of the big advantages of Contentful is supporting those custom needs. I would rephrase this to instead say something more like "They are useful for small demos, but do not allow any customization of the content model. We will cover custom templates in the next section."

File: docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+          "myText": "some fancy text"
+        }
+      }
+    ]
+  },
+  "assets": []
+}
+~~~
+
+Once with the template saved to a file, in this case lets say `templates/simple.json`, you can call the command like follows:
```
Comment (grncdr) [inline docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: This sentence is a bit convoluted, how about "After saving this template to a file named `templates/simple.json` you can call the bootstrap command as follows:"

File: docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+~~~
+
+You may replace `blog` for the other two templates.
+
+##### JSON Templates
+
+The real power of `contentful_bootstrap` comes in the form of JSON Templates.
+Using this tool along with JSON Templates, you can store blueprints for your Spaces.
+
+A JSON Template is a simplified representation of a Space and all of it's contents.
```
Comment (eugenekudashev) [inline docs/tutorials/ruby/using-contentful-bootstrap-for-keeping-up-with-your-spaces.md]: _its_ contents, not _it's_
